tests/configuration/pages/pmmInventoryPage.js

```diff
@@ -1,5 +1,6 @@
-const { I, pmmInventoryPage, inventoryAPI } = inject();
+const { I, inventoryAPI } = inject();
 const assert = require('assert');
+const paginationPart = require('./paginationFragment');
```

Comment:
that's the main idea that part of the page should not be used separately and should not be accessible and registered globally since it's representing only page section: `pmmInventoryPage.pagination.xxx`

Reply:
approach is slightly different: interaction with page fragment require injection the fragment, also fragment is accessed and globally visible without the page. Fragments requires to be added into the helper.
I see the following benefits:
* Given approach does not require updating config helper and registering fragments globally.
* Fragments could be accessed only with the parent page - which is more follows page object concept and OOP. 
* less injections required.